stock_archive_constraint/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Stock archive constraint",
+    "version": "12.0.1.0.0",
+    "license": "LGPL-3",
```

Comment:
Why LGPL? That's only for website related stuff
```suggestion
    "license": "AGPL-3",
```